test/cog/rule_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule RuleTest do
   alias Cog.Models.Rule
 
   setup do
-    bundle = Bundle.changeset(%Bundle{}, %{name: "test_bundle", config_file: %{}, manifest_file: %{}}) |> Repo.insert!
+    bundle = Bundle.changeset(%Bundle{}, %{name: "test_bundle", version: "0.0.1", config_file: %{}, manifest_file: %{}}) |> Repo.insert!
```

Comment:
We don't actually need the manifest_file bit. That was removed earlier. Just missed removing from some tests. If you want to remove it cool, if not, we can do it later.